client/client-lib/src/clients/gateway.rs

```diff
@@ -343,7 +343,12 @@ impl GatewayClient {
     }
 
     pub fn list_fetchable_coins(&self) -> Vec<OutPoint> {
-        self.mint_client().list_active_issuances()
+        let (outpoints, _): (Vec<_>, Vec<_>) = self
```

Comment:
Wait, that can be a `.map(|(out_point, _)| out_point).collect()` instead of the `unzip`, right?

Reply:
oh yes..I was just happy when the compiler didn't scream at me anymore but I also felt like this could be done better xd